### PR TITLE
Add a CI task that checks performance regression

### PR DESCRIPTION
**Pull request by ToSeven**

### CODE REVIEW

File: scripts/executions-check.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+git_checkout(){
+	git reset --hard HEAD~1		
+}
+
+build_run_fuzzer(){
+	./target/release/libafl_cxx ./harness.cc libpng-1.6.37/.libs/libpng16.a -I libpng-1.6.37/ -o fuzzer_libpng -lz -lm || echo "ERROR: Failed to build the fuzzer"
+
+	./fuzzer_libpng > log.txt &
+	timeout 5s ./fuzzer_libpng >/dev/null 2>&1 &
```
Comment (tokatoka) [inline scripts/executions-check.sh:22]: let's make it longer, like 60 sec

Comment (ToSeven) [reply scripts/executions-check.sh]: I have changed 5s to 5m in the latest commit.

**Comment by ToSeven** [comment]
image

Pkill has an illegal option on the macOS platform.

**Comment by tokatoka** [comment]
`killall fuzzer_libpng`?

**Comment by ToSeven** [comment]
Yes, Killing all fuzzer_libpng is no problem. This bug is that the second fuzzer has run before the first fuzzer does not become a Broker, so the first fuzzer has to sleep for a second.  I have fixed this bug on the latest commit https://github.com/ToSeven/LibAFL/commit/0cffe78d9313029e4eb172169cca32e15f74ad9f. @tokatoka

**Comment by tokatoka** [comment]
can you open another PR?

Also can you check your email & discord? You should've received messages from me

**Comment by ToSeven** [comment]
Okay， I have opened a new PR.

**Pull request merged**